src/Plugins/ComplexCore/docs/ResampleImageGeomFilter.md

```diff
@@ -6,19 +6,35 @@ Sampling (Resample)
 
 ## Description ##
 
-This **Filter** changes the **Cell** spacing/resolution based on inputs from the user. The values entered are the desired new spacings (not multiples of the current resolution).  The number of **Cells** in the volume will change when the spacing values are changed and thus the user should be cautious of generating "too many" **Cells** by entering very small values (i.e., very high resolution). Thus, this **Filter** will perform a down-sampling or up-sampling procedure.  
+This **Filter** changes the **Cell** spacing/resolution based on inputs from the user. There are several resampling modes:
+
+### Spacing ###
+The values entered are the desired new spacings (not multiples of the current resolution).  The number of **Cells** in the volume will change when the spacing values are changed and thus the user should be cautious of generating "too many" **Cells** by entering very small values (i.e., very high resolution).
+
+### Scale Factor ###
+The values entered are the desired scaling factor for each dimension, in percentages.  (50%, 50%, 50%) would resample the geometry by half in all dimensions.
+
+### Exact Dimensions ###
+The values entered are the desired cell dimensions of the resampled geometry.  (100, 100, 100) would resample the geometry so that there are 100 cells in each dimension.
```

Comment:
How would this effect the scaling values in each direction. Give a concrete example.